modules/wizard/wizard.php

```diff
@@ -427,27 +427,27 @@ public function add_step_languages( $steps ) {
-				'i18n_language_already_added' => __( 'You already added this language.', 'polylang' ),
-				'i18n_no_language_added'      => __( 'You need to add at least one language.', 'polylang' ),
-				'i18n_add_language_needed'    => __( 'You selected a language, however, to be able to continue, you need to add it.', 'polylang' ),
-				'i18n_pll_add_language'       => __( 'Impossible to add the language.', 'polylang' ),
-				'i18n_pll_invalid_locale'     => __( 'Enter a valid WordPress locale', 'polylang' ),
-				'i18n_pll_invalid_slug'       => __( 'The language code contains invalid characters', 'polylang' ),
-				'i18n_pll_non_unique_slug'    => __( 'The language code must be unique', 'polylang' ),
-				'i18n_pll_invalid_name'       => __( 'The language must have a name', 'polylang' ),
-				'i18n_pll_invalid_flag'       => __( 'The flag does not exist', 'polylang' ),
-				'i18n_dialog_title'           => __( "A language wasn't added.", 'polylang' ),
-				'i18n_dialog_yes_button'      => __( 'Yes', 'polylang' ),
-				'i18n_dialog_no_button'       => __( 'No', 'polylang' ),
-				'i18n_dialog_ignore_button'   => __( 'Ignore', 'polylang' ),
-				'i18n_remove_language_icon'   => __( 'Remove this language', 'polylang' ),
+				'i18n_no_language_selected'   => esc_html__( 'You need to select a language to be added.', 'polylang' ),
+				'i18n_language_already_added' => esc_html__( 'You already added this language.', 'polylang' ),
+				'i18n_no_language_added'      => esc_html__( 'You need to add at least one language.', 'polylang' ),
+				'i18n_add_language_needed'    => esc_html__( 'You selected a language, however, to be able to continue, you need to add it.', 'polylang' ),
+				'i18n_pll_add_language'       => esc_html__( 'Impossible to add the language.', 'polylang' ),
+				'i18n_pll_invalid_locale'     => esc_html__( 'Enter a valid WordPress locale', 'polylang' ),
+				'i18n_pll_invalid_slug'       => esc_html__( 'The language code contains invalid characters', 'polylang' ),
+				'i18n_pll_non_unique_slug'    => esc_html__( 'The language code must be unique', 'polylang' ),
+				'i18n_pll_invalid_name'       => esc_html__( 'The language must have a name', 'polylang' ),
+				'i18n_pll_invalid_flag'       => esc_html__( 'The flag does not exist', 'polylang' ),
+				'i18n_dialog_title'           => esc_html__( "A language wasn't added.", 'polylang' ),
+				'i18n_dialog_yes_button'      => esc_html__( 'Yes', 'polylang' ),
+				'i18n_dialog_no_button'       => esc_html__( 'No', 'polylang' ),
+				'i18n_dialog_ignore_button'   => esc_html__( 'Ignore', 'polylang' ),
+				'i18n_remove_language_icon'   => esc_html__( 'Remove this language', 'polylang' ),
 			)
 		);
 		wp_enqueue_script( 'pll-wizard-languages' );
 		wp_enqueue_style( 'pll-wizard-selectmenu', plugins_url( '/css/selectmenu' . $this->get_suffix() . '.css', POLYLANG_FILE ), array( 'dashicons', 'install', 'common', 'wp-jquery-ui-dialog' ), POLYLANG_VERSION );
```

Comment:
The whole register / localize / enqueue scripts process could be encapsulated into a custom function. So we could automatically escape every script we 'localize'.